docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,162 @@
+:::
+
+### Kconfig.shield
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
+
+### Kconfig.defconfig
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the dongle the keyboard name.
+
+```kconfig
+if SHIELD_MY_BOARD_DONGLE
+
+config ZMK_KEYBOARD_NAME
+    default "My Board"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+endif
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+config ZMK_SPLIT
+    default y
+
+endif
+```
+
+### Dongle configuration
+
+Finally, we will define the dongle configuration. We want to set the number of peripherals to 2 and disable sleep mode for the dongle since it will be connected to the host device.
+In case you want to use the dongle with a battery, you can enable sleep mode for the dongle. Since the dongle will be connected to the host device, we will also increase the broadcast power.
+
+```ini title="my_board_dongle.conf"
+CONFIG_ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS=2
+
+CONFIG_ZMK_SLEEP=n
+# Uncomment if sleep is enabled
+# CONFIG_ZMK_IDLE_SLEEP_TIMEOUT=3600000
+# Make the dongle broadcast with more power
+CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
+```
+
+### Dongle Overlay
+
+In most common cases the dongle will not have any keys, in that case we can instead use a mock kscan module to simulate the keys.
+
+```dts title="my_board_dongle.overlay"
+
+/ {
+    chosen {
+        zmk,kscan = &mock_kscan;
+        // zmk,kscan = &kscan0;
+    };
+
+    mock_kscan: kscan_0 {
+        compatible = "zmk,kscan-mock";
+        columns = <0>;
+        rows = <0>;
+        events = <0>;
+    };
+};
+```
+
+### Building the firmware
+
+After writing the configuration files, you can modify the `build.yml` file to include the dongle configuration.
+
+```yaml
+---
+include:
+  - board: nice_nano_v2
+    shield: my_board_left
+  - board: nice_nano_v2
+    shield: my_board_right
+  - board: nice_nano_v2
+    shield: my_board_dongle
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+After creating your fresh new firmware files, 
+you can go ahead and flash the settings_reset firmware for each half of the keyboard (including the dongle).
+Flash the dongle firmware to the dongle and the left and right firmware to the respective halves.
+
+:::note
+More information about reseting split keyboards can be found under [Troubleshooting Connection Issues](../troubleshooting/connection-issues.mdx#reset-split-keyboard-procedure)
+:::
```

Comment:
I'm not sure you need to refer back to the settings reset twice.

Reply:
Removed double reference